ksp/src/main/kotlin/com/livefront/sealedenum/internal/ksp/SuperInterfaces.kt

```diff
@@ -65,9 +65,9 @@ internal class SuperInterfaces(
             .filter {
                 val declaration = it.declaration
                 declaration is KSClassDeclaration &&
-                        declaration.classKind == ClassKind.INTERFACE &&
-                        declaration.isVisibleInterface() &&
-                        !declaration.isSealedInterface()
+                    declaration.classKind == ClassKind.INTERFACE &&
```

Comment:
Are these changes all required by detekt now?

Reply:
Yeah, it seems like it changed the indention logic a touch